packages/chat-core/src/index.ts

```diff
@@ -148,7 +148,7 @@ interface BotRequest {
     unstructured?: {
       text: string;
     };
-    structured?: StructuredRequest;
+    structured?: StructuredRequest & { slots?: SlotValue[] };
```

Comment:
Why is `slots` optional?

Reply:
I think this might be a question for the backend team. It looks like there's one endpoint which is responsible for a lot of different functionality.

Reply:
Because a structured request can combine `intentId`, `slots` or `choiceId` in the same requests, and all of these are optional (you can combine them every which way).